pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -1,11 +1,23 @@
 # -*- coding: utf-8 -*-
 
+from itertools import chain
+from pkg_resources import resource_string
+import re
+
 import numpy as np
 
 from pyfr.integrators.dual.pseudo.base import BaseDualPseudoIntegrator
 from pyfr.util import proxylist
 
 
+def _get_coefficients(scheme):
+    scheme = re.sub(r'(?<=\))\s*,?\s*(?!$)', r'\n', scheme)
+    a = [[float(f) for f in l.split()] for l in scheme.splitlines() if l]
```

Comment:
I'd group this with the return block.

Reply:
It becomes very messy?

Reply:
Thinking about it, please add some validation code like we have in the BaseTabulatedQuadRule class.  Specifically, have the function take the number of stages as an argument and check the number of lines matches up with what we expect, as does the number of elements in each line.